tests/topotests/bfd_vrflite_topo1/r1/bfd_peers_counter.json

```diff
@@ -0,0 +1,54 @@
+[
+   {
+      "multihop":false,
+      "peer":"192.168.0.2",
+      "vrf":"vrf1",
+      "echo-packet-input":0,
+      "echo-packet-output":0,
+      "session-up":1,
+      "session-down":0,
+      "zebra-notifications":1
```

Comment:
I think we should ignore counters otherwise if they change for some reason (e.g. extra zebra notification or session flap) it will make the test fail.

Reply:
actually, I check there is no more than one session-up; this is one way to check that the udp packets are received accordingly; otherwise I would have session-up and session-down counters increasing (at least this is my guess).

Reply:
ok